src/style.scss

```diff
@@ -71,6 +75,247 @@ rect {
     }
   }
 
+  margin-top: 50px;
+
+}
+
+@keyframes color_change {
+  from { background-color: transparent; }
+  to { background-color: $hover-color; }
+
+}
+button {
+  display: inline-block;
+  margin-bottom: 0;
+  font-weight: normal;
+  text-align: center;
+  vertical-align: middle;
+  touch-action: manipulation;
+  cursor: pointer;
+  background-image: none;
+  border: 1px solid transparent;
+  white-space: nowrap;
+  padding: 6px 12px;
+  font-size: 14px;
+  line-height: 1.42857;
+  border-radius: 4px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+}
+  button:focus, button.focus, button:active:focus, button:active.focus, button.active:focus, button.active.focus {
+    outline: 5px auto -webkit-focus-ring-color;
+    outline-offset: -2px;
+
+  }
+
+button:active, button.active {
+    outline: 0;
+    background-image: none;
+    -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+    //box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+
+  }
+
+
+.btn-removed {
```

Comment:
You can use SCSS (Sass) syntax with nesting:

```
.btn-removed {
  color: white;
  background-color: #f08e65;
  border-color: white;

  &.hover,
  &:active,
  &:focus,
  &:hover {
    background-color: #f56527;
    border-color: #adadad;
  }
}
```

Avoid duplicates, i.e. override only styles that should change.